tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/MigrateCommandTest.php

```diff
@@ -2,28 +2,30 @@
 
 namespace Doctrine\DBAL\Migrations\Tests\Tools\Console\Command;
 
-use Doctrine\DBAL\Migrations\Migration;
 use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use Doctrine\DBAL\Migrations\Migration;
 use Doctrine\DBAL\Migrations\Tools\Console\Command\MigrateCommand;
-use Symfony\Component\Console\Helper\HelperSet;
-use Symfony\Component\Console\Helper\QuestionHelper;
-use Symfony\Component\Console\Input\ArrayInput;
 
 class MigrateCommandTest extends CommandTestCase
 {
     use DialogSupport;
 
-    const VERSION = '20160705000000';
+    public const VERSION = '20160705000000';
 
+    /** @var Migration */
     private $migration;
 
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->configureDialogs($this->app);
+    }
+
     public function testPreviousVersionErrorsWhenThereIsNoPreviousVersion()
     {
         $this->willResolveVersionAlias('prev', null);
 
-        list($tester, $statusCode) = $this->executeCommand([
-            'version' => 'prev',
-        ]);
+        list($tester, $statusCode) = $this->executeCommand(['version' => 'prev']);
```

Comment:
Not sure if there's a sniff for it, but what do you think of using short syntax for `list` assignments?
`[$tester, $statusCode] = $this->executeCommand(['version' => 'prev']);`

Reply:
There is no sniff for this, but it'd be cool to have one. I changed these manually for now.

Reply:
I'll see if I can come up with something - shouldn't be too hard 👍